WORKSPACE

```diff
@@ -141,8 +141,17 @@ apple_support_dependencies()
 ##############################################################################
 # Java
 ##############################################################################
+# Java microgenerator.
+_java_microgenerator_version = "3f098c4e21daa7b211cd43e6c4af03d627e754e4"  # Will change often in dev.
```

Comment:
Please don't use variables in workspace file. This lets to save the one duplicate occurrence of this git hash, but it  converts the workspace file from being declarative to being imperative. It is not a big deal here, but it opens a door for treating this WORKSPACE file as an "program", complicating this already very hard to parse file.